gulpfile.js

```diff
@@ -185,15 +186,34 @@ function enableDemo(done) {
     done();
 }
 
+function inspect() {
+    var nodeInspector = require('gulp-node-inspector');
+    return gulp.src(locationConfig.server.filesForBuild)
+    .pipe(nodeInspector({
+          webPort: 8081
+        }));
+
+}
 function startServer(done) {
+    getServer(false, done)
+}
+
+function startServerDebug(done) {
```

Comment:
You don't need a separate function for this. Just use the args.dev flag inside the original startServer